src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceConditionService.kt

```diff
@@ -210,7 +362,33 @@ class LicenceConditionService(
     additionalConditionRepository.saveAndFlush(updatedAdditionalCondition)
 
     val username = SecurityContextHolder.getContext().authentication.name
+    val createdBy = communityOffenderManagerRepository.findByUsernameIgnoreCase(username)
+      ?: throw RuntimeException("Staff with username $username not found")
+
     val updatedLicence = licenceEntity.copy(dateLastUpdated = LocalDateTime.now(), updatedByUsername = username)
     licenceRepository.saveAndFlush(updatedLicence)
+
+    val changes = mapOf(
+      "typeOfChange" to "update",
+      "condition" to "additional data",
+      "changes" to listOf(
+        mapOf(
+          "conditionCode" to updatedAdditionalCondition.conditionCode,
+          "conditionType" to updatedAdditionalCondition.conditionType,
+          "conditionText" to updatedAdditionalCondition.expandedConditionText
+        )
+      ),
+    )
```

Comment:
Similar to bespoke, am I right in thinking that this won't show a before-and-after of what the changes are?